casa/Logging/LogOrigin.h

```diff
@@ -171,9 +175,9 @@ class LogOrigin
     String task_p;
     String function_p;
     String class_p;
-    ObjectID id_p;
-    uInt line_p;
-    String file_p;
+    ObjectID id_p = True;
+    uInt line_p = 0;
+    String file_p = "";
```

Comment:
It's odd that you don't default-initialise `task_p`, `function_p`, `class_p` and `node_p` but you do it for `file_p`.
```suggestion
    String file_p;
```